src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString.split(';');
+  const result = {};
+
+  arr.forEach((style) => {
+    if (style.trim()) {
+      const [key, value] = style.split(':');
+
+      result[key.trim()] = value.trim();
```

Comment:
You can simplify like this
```suggestion
      const [key, value] = style.split(':').map(item => item.trim());

      result[key] = value;
```